### DISCUSSION TITLE
运行HelloVitalik报错

### DISCUSSION THREAD

**Discussion by hanqihong** [AmazingAng/WTF-Solidity#500] [General]
[Running] node "e:\nodejs\hello.js"
file:///e:/nodejs/hello.js:5
    console.log(`ETH Balance of vitalik: ${ethers.utils.formatEther(balance)} ETH`);
                                                        ^

TypeError: Cannot read properties of undefined (reading 'formatEther')
    at main (file:///e:/nodejs/hello.js:5:57)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.15.0

**Comment by Azleal** [comment]
首先，你只发报错没人能知道是什么原因导致的。
其次，盲猜一下你用的`ethers`版本和你的用法不匹配. v6中没有`utils`了

**Comment by Zhuoli** [comment]
代码改成 ethers.formatEther 就可以了  谢谢楼上提示

**Comment by LBJyang** [comment]
改成ethers.formatEther，查一下官方文档。
